database/engine/README.md

```diff
@@ -101,14 +108,34 @@ and streaming recipient nodes):
 
 An important observation is that RAM usage depends on both the `page cache size` and the `dbengine disk space` options.
 
+### Example:
+
+Let's say we have 1 Netdata Master Node with 10 slaves. I want to save the data for 1 week without downsampling, and afterwards I will downsample to 1 sample per 5 minutes.
+
+For the archiving I will be using the Backends functionality, so we wont't elaborate any further here.
+
+For the one week's worth of data, I will have 10 DB engine instances for the *Slaves* and 1 for the *Master*.
+
+Each Slave saves 2000 metrics/s, and Master Node only 1000/s.
+
+Thus, taking into account that for 4000 metrics/s the default settings will save 1 day, with 2000 metrics/s I will save for 2 days.
+
+Final Disk Space = (256 * 10 slaves * 7 days ) / 2 + (256 * 7 * 1 Master) / 4 MiB
+
+Final RAM usage = (10 slaves + 1 Master)* (32 * 1.049 * 10^6) Page cache size + (2000 dimensions * 10 slaves * 4096 * 2 ) + (#pages-on-disk * 4096 * 0.03) + (1000 dimensions * 1 Master * 4096 * 2 ) + (#pages-on-disk * 4096 * 0.03) bytes
```

Comment:
A few comments on this new section:

We don't use "I" in our documentation—these pronouns should be changed to "you," as though the reader is the one setting up this configuration and trying to understand memory usage.

I wouldn't capitalize words like _Slaves_, _Master_, or _Node_.

I'd like to see some example numbers from those last two calculations. Asking the reader to get out a calculator to do it themselves is too complex. I also wonder if there's a way to present the numbers in a clearer fashion. Even putting them into a code block would help make it more readable.

Reply:
I will make it more readable. 

Since we are discussing about this, can we see how to compute the variable #pages-on-disk, since I was unable to find it. Moreover, as a user, I would like to read about the correlation between `disk space`-`page cache size` and metrics retention period.

At the moment, I can only proceed with rule of 3 (i.e if I want double the metrics, I need to double both variables for the same retention period)